### ISSUE TITLE
Translate the guide into other languages

### ISSUE THREAD

**Issue by gerzenstl** [brandonskerritt/HowToGetAnyJobYouWant#23]
Thanks for this guide.
Have you ever considered to add option to translate into other languages?

**Comment by bee-san** [comment]
Yes, I have! And an audio book too. Problem is that I don't know any other languages, & audio books are hard. 

Any thoughts on what language? I'll check it out :)

**Comment by gerzenstl** [comment]
Since I speak Spanish, I could help with that translation.
